playbooks/common/openshift-cluster/upgrades/v3_1_minor/pre.yml

```diff
@@ -41,7 +41,7 @@
       g_new_service_name: "{{ 'origin' if deployment_type =='origin' else 'atomic-openshift' }}"
 
   - name: Determine available versions
-    script: ../files/versions.sh {{ g_new_service_name }} openshift
+    script: ../files/versions.sh {{ g_new_service_name }}
```

Comment:
I think this attempted to handle cases where they're upgrading within 3.0.x. The `sort -r` in versions.sh should make this a non issue assuming that atomic-openshift version is higher than that of the available openshift version. I think this is safe to remove at this point so +1